feathernotes/fn.cpp

```diff
@@ -1315,6 +1315,9 @@ void FN::fileOpen (const QString &filePath, bool startup, bool startWithLastFile
         if (file.open (QIODevice::ReadOnly))
         {
             QTextStream in (&file);
+#ifdef __OS2__
+            in.setCodec(QTextCodec::codecForName("UTF-8"));
```

Comment:
Then I have to fork here. QT6 for OS/2 is on the horizon, but not really usable at the moment.
Without this fix, you cannot read in a file without destroying the text.

Reply:
> Without this fix, you cannot read in a file without destroying the text.

I'm sorry about that. You might want to use a simple build patch for OS/2, if this is the only problem there.

Qt's removal of support for legacy encodings was inevitable — some dves think it should have been done sooner. Everything is supposed to be in UTF-8, and automatic unicode detection is enabled. This fixes a mess that had existed for a long time, but it may be uncomfortable under rare circumstances.